includes/defaults.inc.php

```diff
@@ -80,12 +80,12 @@
 // $config['rrdcached']    = "unix:/var/run/rrdcached.sock";
 
 // Web Interface Settings
-if (isset($_SERVER['SERVER_NAME']) && isset($_SERVER['SERVER_PORT'])) {
-    if (strpos($_SERVER['SERVER_NAME'], ':')) {
+if (isset($_SERVER['HOST_NAME']) && isset($_SERVER['SERVER_PORT'])) {
```

Comment:
$_SERVER['HOST_NAME'] is not a valid variable, so you have effectively deleted all this code afaict.  What do the different variables contain on your server?

Reply:
Right, sorry. It seems I didn't quite pay attention when copying my tested code over to my cloned repo. I used HTTP_HOST, not HOST_NAME.